ledger/src/snapshot_utils.rs

```diff
@@ -32,7 +33,53 @@ pub const TAR_SNAPSHOTS_DIR: &str = "snapshots";
 pub const TAR_ACCOUNTS_DIR: &str = "accounts";
 pub const TAR_VERSION_FILE: &str = "version";
 
-pub const SNAPSHOT_VERSION: &str = "1.1.0";
+const MAX_SNAPSHOT_DATA_FILE_SIZE: u64 = 32 * 1024 * 1024 * 1024; // 32 GiB
+const VERSION_STRING_V1_1_0: &str = "1.1.0";
+const VERSION_STRING_V1_1_1: &str = "1.1.1";
```

Comment:
Might as well call this 1.2.0, since it'll hopefully ride the 1.2.x release train
```suggestion
const VERSION_STRING_V1_1_1: &str = "1.2.0";
```

Reply:
Yeah, let's be bold. And bump to `1.2.0`. :)

Reply:
Version string renumbered to 1.2.0.

Variable / Type / Constant names too !